Static/DataSource.swift

```diff
@@ -275,12 +293,16 @@ extension DataSource: UITableViewDelegate {
         if let row = row(at: indexPath) {
             row.selection?()
         }
+
+        tableViewDelegate?.tableView?(tableView, didSelectRowAt: indexPath)
```

Comment:
why is this needed if we have the forwarding target?.. wouldn't it send this there anyway?.. I may be misunderstanding forwardingTarget...

Reply:
Since `DataSource` implements this method, it gets the message sent to it, and thus it doesn't go to the forwardingTarget.

Messages only get forwarded to the forwardingTarget if this object doesn't implement.